vernac/vernacentries.ml

```diff
@@ -741,8 +747,8 @@ let private_ind =
   | None -> return false
 
 let vernac_inductive ~atts kind indl =
-  let (template, (poly, cumulative)), private_ind = Attributes.(
-      parse Notations.(template ++ polymorphic_cumulative ++ private_ind) atts) in
+  let ((template, (poly, cumulative)), private_ind), typing_flags = Attributes.(
+      parse Notations.(template ++ polymorphic_cumulative ++ private_ind ++ typing_flags) atts) in
```

Comment:
We accept typing flags but record and definitional classes ignore it here

Reply:
Ok will fix.

Reply:
I'd like to push the record refactoring before propagating this flag to record so I've made it an error.